plugins/apps/commands

```diff
@@ -20,17 +20,19 @@ case "$1" in
     [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
     [[ ! -d "$DOKKU_ROOT/$2" ]] && echo "App $2 does not exist" && exit 1
     [[ "$2" == "tls" ]] && echo "Unable to destroy tls directory" && exit 1
-    APP="$2"; IMAGE="dokku/$APP";
+    APP="$2"; IMAGE="dokku/$APP"; FORCE="$3"
```

Comment:
Having to `echo app_name | dokku apps:destroy app_name` was annoying. This allows `dokku apps:destroy app_name force`